recipes/openmesh/bld.bat

```diff
@@ -0,0 +1,24 @@
+rem  @echo off
+
+mkdir build_win
+cd build_win
+
+set CMAKE_CONFIG="Release"
+
+cmake -LAH -G"NMake Makefiles"                               ^
+  -DCMAKE_FIND_ROOT_PATH="%LIBRARY_PREFIX%"                  ^
+  -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"                  ^
+  -DBUILD_APPS=OFF                                           ^
+  -DBoost_LIBRARIES="C:/Miniconda/envs/_build/Library/lib/boost_python-vc90-mt-1_61.lib" ^
```

Comment:
You can probably use `BOOST_ROOT` rather than hardcoding this path - at the very least you can use `%LIBRARY_PREFIX%`